### ISSUE TITLE
Sequence extraction of spite reads and discordants reads

### ISSUE THREAD

**Issue by caijingtao1993** [iprada/Circle-Map#44]
Dear Inigo:
       Hello! About Circle-Map, I use Circle-Map version=1.1.3 to identify eccDNA, Circle-Map Realign command to get the result, I want to get the sequence of the spite reads of each eccDNA in the sample and the sequence of Discordants reads, is there any instruction or way ? I check the Circle-Map ReadExtractor parameters, and I can get Reads indicating circular DNA structural variants. Discordants reads and spite reads cannot be output directly. Is there a way to extract the two types of read sequences corresponding to eccDNA based on the *_sort_circular_read_candidates.bam file? Looking forward to your reply.Would like to ask you ! Many thanks !

**Comment by iprada** [comment]
Hi @caijingtao1993,

Thanks for posting it here. You mean you want to get exact read used for calling a circle?

Best,

Inigo

**Comment by caijingtao1993** [comment]
> 嗨@ caijingtao1993，
> 
> 感谢您在此处发布。您的意思是说您想获取用于呼叫圈子的准确阅读信息吗？
> 
> 最好，
> 
> Inigo

Yes, according to Circle-Map Realign to get eccDNA, I want to get the spite reads of each eccDNA in the sample and the sequence of Discordants reads, generate bam file or fastq file

**Comment by iprada** [comment]
Hi,

I see. I answered a similar question before.

Look at this post https://github.com/iprada/Circle-Map/issues/39

Best,

Inigo

**Comment by huangyizR** [comment]
Hi Inigo,
You mentioned in #39 that using this command to extract the split reads for each ecDNA:

Circle-Map bam2bam -i sorted_circular_dna_candidates.bam -qbam qname.bam -fasta /path/to/hg38.fa -o output.bam 

But I cannot find the command option in the version I used. The version I installed before is 1.1.2 through conda. I wonder that should I update to 1.1.4 to access the bam2bam command? 

Thanks for your time!

**Comment by iprada** [comment]
Hi,

Yes, you should upgrade it to v1.1.4 :)

best,

Inigo
